### PR TITLE
Travis CI

### PR DESCRIPTION
**Pull request by simonseyock**
Make the tests run on travis and use it.

### CODE REVIEW

File: html/client.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 </head>
 
 <body>
-<div id="map" class="map" tabindex="1">
+<div id="mymap" class="map" tabindex="1">
```
Comment (ischas) [inline html/client.html:19]: I don't like the naming with a prefixed "my". Shouldn't we change it to "g4u-map"? The same applies to the configurations.

File: package.json [hunk 2]
```diff
@@ -35,6 +35,7 @@
     "concurrently": "2.2.0",
     "esdoc": "0.4.7",
     "mocha": "2.5.3",
+    "phantomjs-prebuilt": "^2.1.11",
```
Comment (ischas) [inline package.json:38]: Change to a fixed version number? We have greenkeeper active. ;-)

**Comment by ischas** [comment]
We should add a Travis-Badge to readme.md.

**Comment by simonseyock** [comment]
I agree

**Pull request closed without merging**